services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -2894,7 +2894,7 @@ public String getRealTimeTopic(String clusterName, String storeName) {
         if (store == null) {
           throwStoreDoesNotExist(clusterName, storeName);
         }
-        if (!store.isHybrid() && !store.isWriteComputationEnabled()) {
+        if (!store.isSystemStore() && !store.isHybrid() && !store.isWriteComputationEnabled()) {
           logAndThrow("Store " + storeName + " is not hybrid, refusing to return a realtime topic");
```

Comment:
What is the implication of adding this condition? System stores are allowed to be not fully initialized...?

Reply:
Yeah. Let me rethink about this.

Reply:
I think  I'll address meta store and push status store creation issues in a follow up PR as it might take some time and refactoring.